pkg/zsio/zson_test.go

```diff
@@ -82,16 +82,22 @@ const zson3 = `
 #0:record[foo:set[string]]
 0:[[-;]]`
 
-// string \x2d is "-"
+// String \x2d is "-".
 const zson4 = `
 #0:record[foo:string]
 0:[\x2d;]`
 
-// string \x5b is "[", second string is "[-]" and should pass through
+// String \x5b is "[", second string is "[-]" and should pass through.
 const zson5 = `
 #0:record[foo:string,bar:string]
 0:[\x5b;\x5b-];]`
 
+// Make sure we handle unset fields and empty sets.
+// Henri found these bugs in the zjson PR.
```

Comment:
Honored to be cited here but is it relevant?!

Reply:
ok, I took it out